kubernetes/assets/dashboards/kubernetes_pods.json

```diff
@@ -75,22 +75,22 @@
                 "type": "timeseries",
                 "requests": [
                     {
-                        "q": "sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:failed,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:pending,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:running,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.pod.status_phase{$cluster,$namespace,$deployment,pod_phase:succeeded,$scope,$statefulset,$daemonset,$job}",
+                        "q": "sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:failed,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:pending,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:running,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.pod.status_phase{$kube_cluster_name,$kube_namespace,$kube_deployment,pod_phase:succeeded,$scope,$kube_stateful_set,$kube_daemon_set,$job}",
```

Comment:
There's actually a limitation there in the ksm check: we don't have the kube_deployment, kube_stateful_set, or kube_daemonset tags. We should just remove those dimensions because they won't work for now. The KSM v2 check will have them and we'll be able to add them back when it's in use.